dags/snowflake_examples/taxi_snowflake_el.py

```diff
@@ -0,0 +1,294 @@
+from airflow import DAG
+from airflow.decorators import task
+from airflow.models import Variable
+from airflow.models.baseoperator import chain
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.utils.dates import datetime
+from airflow.utils.task_group import TaskGroup
+from airflow.providers.amazon.aws.transfers.local_to_s3 import LocalFilesystemToS3Operator
+from airflow.providers.snowflake.operators.snowflake import SnowflakeOperator
+from airflow.providers.snowflake.transfers.s3_to_snowflake import S3ToSnowflakeOperator
+from plugins.snowflake_check_operators import (
+    SnowflakeCheckOperator,
+    SnowflakeValueCheckOperator,
+    SnowflakeIntervalCheckOperator,
+    SnowflakeThresholdCheckOperator,
+)
+
+import pandas as pd
+
+# This table variable is a placeholder, in a live environment, it is better
+# to pull the table info from a Variable in a template
+TABLE = "YELLOW_TRIPDATA"
+DATES = ["2019-01", "2019-02"]
+TASK_DICT = {}
+
+with DAG(
+    "taxi_snowflake_el",
+    start_date=datetime(2021, 7, 7),
+    description="A sample Airflow DAG to perform data quality checks using SQL Operators.",
+    schedule_interval=None,
+    default_args={
+        "snowflake_conn_id": "snowflake_default",
+        "warehouse": Variable.get("SNOWFLAKE_WH"),
```

Comment:
what's the purpose of putting the snowflake parameters in Variables instead of in the connection object?

Reply:
Because I forgot you can pull them from the connection object 😓